css/variables.scss

```diff
@@ -1,32 +1,32 @@
-$input-width: 130px;
+$input-width: 130px !default;
 
-$color-babu: #00a699;
-$color-babu-dark: #00514a;
-$color-white: #fff;
+$color-babu: #00a699 !default;
+$color-babu-dark: #00514a !default;
+$color-white: #fff !default;
 
-$border-color: #dbdbdb;
-$placeholder-color: #757575;
-$text-color: #484848;
-$focus-color: #99ede6;
-$focus-text-color: #007a87;
+$border-color: #dbdbdb !default;
```

Comment:
If we are renaming variables, it would also be nice to apply a consistent naming scheme. I like names that go from most general on the left to most specific on the right. e.g. `$namespace-type-name` e.g. `$react-dates-color-babu` is good. In this case, that would look like `$react-dates-color-border`, `$react-dates-color-placeholder`, `$react-dates-color-text-focus`, `$react-dates-border-chevron` etc.